assets/app/view/game/corporation.rb

```diff
@@ -338,10 +338,13 @@ def entities_rows(entities)
           type = entity.player? ? 'tr.player' : 'tr.corp'
           type += '.bold' if last_acted_upon
           name = entity.player? ? entity.name : "© #{entity.name}"
+          show_percent = @game.instance_variable_get(:@show_share_percent_ownership)
```

Comment:
@game.class::SHOW_SHARE_PERCENT_OWNERSHIP

Reply:
Thanks, couldn't figure out how to access the class property.  kept getting undefined method error...